schemas/sources.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE data.covid_sources (
```

Comment:
See comment about whether this should be called `covid_sources` or just `sources` (to leave room for expansion)